_pytest/main.py

```diff
@@ -90,10 +90,12 @@ def wrap_session(config, doit):
             session.exitstatus = doit(config, session) or 0
         except pytest.UsageError:
             raise
-        except KeyboardInterrupt:
+        except KeyboardInterrupt as e:
             excinfo = _pytest._code.ExceptionInfo()
```

Comment:
Few points:
- there's no need for `as e` because the exception data is already in `excinfo`
- I'm unsure if this is the right place to do the outputting
- I tested `pytest.exit`ing from a test, and it now prints the msg twice - I think investigating this and why it doesn't print when `pytest.exit`ing from `pytest_configure` will point to the right place to do the outputting